website/src/pages/reset-password.page.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+import Layout from "src/layout"
+import { centeredColumn } from "src/sprinkles.css"
+import { centeredForm, loginFormBox, positionButton, skinnyWidth, submitButton } from "./login.css";
+
+// TODO: this page is incomplete
+const ResetPasswordPage = () => {
+
+    const [email, setEmail] = useState("");
+
+    return (<Layout>
+        <main className={skinnyWidth}>
```

Comment:
Mind running `prettier` on the files you've changed? I use the format-on-save functionality in my editor so I never have to worry about it.